File: src/qiu_1cpp_pkg/src/person.cpp
#include"rclcpp/rclcpp.hpp"
#include<string>
#include<iostream>
using namespace std;
class person
{
    public:
        person(const string& name,int age):m_name(name),m_age(age){}

        void havefun(const string& game)
        {
            cout<<"我是 "<<m_name<<"我喜欢 "<<game<<endl;
        }
    private:
        string  m_name;
        int m_age;

};

int main(int argc,char**argv)
{

    person p("kunkun",11);
    p.havefun("math");
    return 0;
}

File: src/qiu_second_node_pkg/qiu_second_node_pkg/person.py
class person:
    def __init__(self,name,age) -> None:
        self.name = name
        self.age = age
    def havefun(self,game):
        print(f"我是{self.name},我已经{self.age}岁了，我喜欢玩{game}")


def main():
    qiu = person("kunkun",14)
    qiu.havefun("math")
